test/e2e/fixture/cluster/actions.go

```diff
@@ -52,7 +54,8 @@ func (a *Actions) Create(args ...string) *Actions {
 	})
 
 	if err != nil {
-		log.Fatalf("Failed to upser cluster %v", err.Error())
```

Comment:
In the past, we've discovered that is it better to fail by default and change that behavior using something like `IgnoreErrors(true)` explicitly. The problem is that command might fail when we don't expect it to fail. So the test eventually fails but it is really difficult to say why because actually error is hidden.